docs/build/get-started.md

```diff
@@ -322,66 +324,59 @@ pub fn hello_world() -> String {       // 4
 }
 ```
 
-Before we do anything, (1) we need to import the [Marine Rust SDK](/docs/marine-book/marine-rust-sdk/marine-rust-sdk.md),
-which allows us to compile Rust code to wasm32-wasi module compatible with Fluence’s Marine runtime. The `#[marine]` macro, (3), is part of the *marine-rust-sdk* and exports marked types as publicly visible and callable functions and structs. In (4) we implement our business logic, which ain’t much this time around.
+But we don't yet have a place to put that code. For that, we need to add a Service by using command `fluence service new`.
 
-In (2), we implement a main function which is not marked with the *#[marine]* procedural macro.
-We discuss modules and module configuration further below. Also note that WASM IT has type limits,
-which are explained in detail in the [Marine book](/docs/marine-book/marine-runtime/i-value-and-i-type).
-The short version is: you got strings, ints, floats, bytes, arrays and records at your disposal,
-but you do not have generics, lifetimes, etc.
+Let’s unbundle this command before we follow the prompts: As discussed earlier, you write your business logic in Rust and compile it to one or more Wasm modules. You then “package” these modules, with help of Fluence CLI, into a *service*. Eventually you deploy this service to one or more peers and use Aqua to interact with the deployed service(s).
+
+If your business logic results in only a single module, like our *hello_world* code, then you will have a service with a single module.
 
-Now that we know what our code looks like, let’s use Fluence CLI to scaffold our Rust (sub-)project with the `fluence service new` command. Let’s unbundle this command before we follow the prompts: As discussed earlier, you write your business logic in Rust and compile it to one or more Wasm modules. You then “package” these modules, with help of Fluence CLI, into a *service*. Eventually you deploy this service to one or more peers and use Aqua to interact with the deployed service(s).  If your business logic results in only a single module, like our *hello_world* code, then this module is also the service.
+Now let’s use Fluence CLI to scaffold our Rust service with the `fluence service new` command.
 
-Now we follow the prompts to `fluence service new` and complete the setup:
+Follow the prompts and complete the setup:
 
 ```bash
 fluence service new
-? Enter service path service
-? Do you want to use service as the name of your new service? No
-? Enter service name (must start with a lowercase letter and contain only letters, numbers, and underscores)
-hello_world
-Successfully generated template for new service at service
+? Enter service path hello_world
```

Comment:
why not just `fluence service new hello_world`?

Reply:
good idea!